.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  GH_TOKEN: ${{ github.token }}      
+
+jobs:
+    release:
+        name: Create new release
+        runs-on: ${{ matrix.os }}
+        strategy:
+          matrix:
+            include:
+              - os: ubuntu-latest
+                filename_suffix: "${{ github.ref_name }}-linux-amd64"
+        steps:
+          - uses: actions/checkout@v4
+          - uses: dtolnay/rust-toolchain@stable
+          - run: cargo build --release --locked ${{ matrix.flags }}
+          - run: gh release create ${{ github.ref }} --draft --title "Uhyve ${{ github.ref_name }}"
+          # Adapted from: https://github.com/mkroening/edu-sync/blob/main/.github/workflows/release.yml
+          - name: Create archive
+            run: |
+              DIST=uhyve-${{ matrix.filename_suffix }}
+              mkdir $DIST
+
+              mv target/release/uhyve $DIST
+              cp README.md $DIST
+              cp LICENSE-* $DIST
+              
+              tar czf $DIST.tar.gz $DIST
+              echo "FILE=$DIST.tar.gz" >> $GITHUB_ENV
+          - run: gh release upload ${{ github.ref }} $FILE --clobber
```

Comment:
`--clobber` was used for testing, right? For releases, we should never override existing assets.

Reply:
Erroneous mistake, sorry!